README.md

```diff
@@ -84,14 +83,13 @@ section of the project wiki.
 
 
 ## Contributing
-Contributions are very welcomed, however please follow the below guidelines.
+Contributions are very welcomed, however, please follow the below guidelines.
```

Comment:
Re-wording this sentence completely might be an option, but as it stands, the comma doesn't belong here. "however" is modifying the latter half of the sentence, not the former.

Reply:
Oh, nice catch again Carson. You're correct about the comma. I looked over this sentence rather quickly but I should have double-checked it.